examples/classifier/CL1_compute_dtw_distance.py

```diff
@@ -0,0 +1,33 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
+import warnings
+warnings.filterwarnings(("ignore"))
```

Comment:
I think instead of filtering the warning implicitly, it is better either modify your codes to get rid of the warnings or just leave the warnings there and let the user decide if they want to filter or not

Reply:
Very true! I am going to remove it. Thanks for noticing!